src/pages/SearchPage/SearchPage.js

```diff
@@ -1,95 +1,40 @@
 import React, { useState, useEffect } from 'react';
-import { useHistory, useParams } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
 import SearchPageWrapper from './SearchPage.style';
-import SearchBar from '../SearchBar/SearchBar';
+import SearchBar from './SearchBar/SearchBar';
 import Spinner from '../../components/Spinner/Spinner';
 
 
 const ONE_YEAR_IN_SECONDS = 60 * 60 * 24 * 365;
 
-async function fetchData(url) {
+async function handleSearch(subreddit) {
+  const oneYearAgo = Math.round((new Date()).getTime() / 1000) - ONE_YEAR_IN_SECONDS;
+  const url = `https://api.pushshift.io/reddit/search/submission/?subreddit=${subreddit}&sort=desc&sort_type=score&after=${oneYearAgo}&size=500`;
```

Comment:
Btw: Not sure if you noticed but I replace the sorting by date with `score`

Reply:
I did, thanks.